### PR TITLE
hwmv2: Convert neorv32 soc/board

### PR DESCRIPTION
**Pull request by nordicjm**

### CODE REVIEW

**Review by gmarull — APPROVED** [review]

**Comment by henrikbrixandersen** [comment]
CI complains: "Board 'neorv32' does not define any revisions"?

File: boards/others/neorv32/board.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+board:
+  name: neorv32
+  vendor: others
+  revision:
+    format: major.minor.patch
+    default: "1.8.6"
+    revisions:
+      - name: "1.0.0"
```
Comment (henrikbrixandersen) [inline boards/others/neorv32/board.yml:8]: What is revision 1.0.0?

Comment (henrikbrixandersen) [reply boards/others/neorv32/board.yml:8]: 1.8.6 is currently the only revision of the IP supported.

Comment (nordicjm) [reply boards/others/neorv32/board.yml]: Fixed

**Review by henrikbrixandersen — APPROVED** [review]

**Pull request merged**